modules/reducers/getInsightSidebar.js

```diff
@@ -0,0 +1,19 @@
+import { get } from 'lodash';
+import AppConstants from '../constants';
+
+const initialSidebarState = {
```

Comment:
@jyash97 Why you're creating a separate reducer for it?
Use the `getAnalyticsInsights.js` reducer and change the name of it to `analyticsInsights.js` everywhere.